src/www/ui/api/Controllers/GroupController.php

```diff
@@ -1,7 +1,7 @@
 <?php
 /***************************************************************
  Copyright (C) 2021 Orange
- Author: Piotr Pszczola <piotr.pszczola@orange.com>
+ Authors: Piotr Pszczola <piotr.pszczola@orange.com> , Samuel Dushimimana <dushsam100@gmail.com>
```

Comment:
Adding yourself to the author list will not be correct. Please add yourself to the copyright list by adding another copyright (`Copyright (C) 2022 <your details>`)